### PR TITLE
Gettextize cli_wallet_daemon_isolation_qubes_whonix.md and howto_fix_stuck_funds.md

### PR DESCRIPTION
**Pull request by erciccione**
This is the first PR which adds `.pot` and `.po` files to be fed to weblate and are located in `_18n/$LANG/resources/user-guides/weblate/$GUIDE.po(t)`. All files have been manually checked and "unfuzzed".

I'm currently testing this PR on Weblate.

Part of #1270

### CODE REVIEW

**Comment by erciccione** [comment]
Tested OK on Weblate.

File: _i18n/en/resources/user-guides/weblate/cli_wallet_daemon_isolation_qubes_whonix.pot [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+"purposes. First you will create a Whonix workstation for the wallet with no "
+"networking. Next, another Whonix workstation for the @daemon which will use "
+"your Whonix gateway as it's NetVM. For communication between the wallet and "
+"daemon you can make use of Qubes [qrexec](https://www.qubes-os.org/doc/"
+"qrexec3/)."
+msgstr ""
+
+#. type: Plain text
+#: _i18n/en/resources/user-guides/cli_wallet_daemon_isolation_qubes_whonix.md:8
+msgid "new paragraph of text that would mess up the entire format."
```
Comment (wirrickelliot) [inline _i18n/en/resources/user-guides/weblate/cli_wallet_daemon_isolation_qubes_whonix.pot:46]: Is this supposed to be here?

Comment (erciccione) [reply _i18n/en/resources/user-guides/weblate/cli_wallet_daemon_isolation_qubes_whonix.pot:46]: No. It's a leftover from my tests. Nice catch :)

File: _i18n/en/resources/user-guides/weblate/cli_wallet_daemon_isolation_qubes_whonix.pot~ [hunk 2]
```diff
@@ -0,0 +1,251 @@
+# Translation file for Weblate
```
Comment (wirrickelliot) [inline _i18n/en/resources/user-guides/weblate/cli_wallet_daemon_isolation_qubes_whonix.pot~:1]: I don't believe this file should be here.

File: _i18n/it/resources/user-guides/weblate/cli_wallet_daemon_isolation_qubes_whonix.po~ [hunk 3]
```diff
@@ -0,0 +1,321 @@
+# Translation files for Weblate
```
Comment (wirrickelliot) [inline _i18n/it/resources/user-guides/weblate/cli_wallet_daemon_isolation_qubes_whonix.po~:1]: I don't believe this file should be here either.

File: _i18n/en/resources/user-guides/weblate/cli_wallet_daemon_isolation_qubes_whonix.pot [hunk 4]
```diff
@@ -0,0 +1,251 @@
+# Translation file for Weblate
+# Copyright (C) 2021,The Monero Project
+# This file is distributed under the same license as the monero-site repository.
+#
+#, fuzzy
```
Comment (wirrickelliot) [inline _i18n/en/resources/user-guides/weblate/cli_wallet_daemon_isolation_qubes_whonix.pot:5]: This should be unfuzzied.

File: _i18n/en/resources/user-guides/weblate/cli_wallet_daemon_isolation_qubes_whonix.pot [hunk 5]
```diff
@@ -0,0 +1,251 @@
+# Translation file for Weblate
+# Copyright (C) 2021,The Monero Project
```
Comment (wirrickelliot) [inline _i18n/en/resources/user-guides/weblate/cli_wallet_daemon_isolation_qubes_whonix.pot:2]: The headers aren't uniform between all of the files.

File: _i18n/it/resources/user-guides/weblate/cli_wallet_daemon_isolation_qubes_whonix.po [hunk 6]
```diff
@@ -0,0 +1,256 @@
...
+"purposes. First you will create a Whonix workstation for the wallet with no "
+"networking. Next, another Whonix workstation for the @daemon which will use "
+"your Whonix gateway as it's NetVM. For communication between the wallet and "
+"daemon you can make use of Qubes "
+"[qrexec](https://www.qubes-os.org/doc/qrexec3/)."
+msgstr ""
+
+#. type: Plain text
+#: _i18n/en/resources/user-guides/cli_wallet_daemon_isolation_qubes_whonix.md:8
+msgid "new paragraph of text that would mess up the entire format."
```
Comment (wirrickelliot) [inline _i18n/it/resources/user-guides/weblate/cli_wallet_daemon_isolation_qubes_whonix.po:45]: This is still present in multiple files.

File: _i18n/fr/resources/user-guides/weblate/cli_wallet_daemon_isolation_qubes_whonix.po [hunk 7]
```diff
@@ -0,0 +1,289 @@
...
+
+#. type: Plain text
+#: en/resources/user-guides/cli_wallet_daemon_isolation_qubes_whonix.md:12
+msgid "+ Using a Whonix workstation template, create two workstations as follows:"
+msgstr ""
+"+ En utilisant un modèle Whonix de station de travail, créez deux stations "
+"de travail comme suit :"
+
+#. type: Bullet: '  - '
+#: en/resources/user-guides/cli_wallet_daemon_isolation_qubes_whonix.md:14, markdown-text
```
Comment (wirrickelliot) [inline _i18n/fr/resources/user-guides/weblate/cli_wallet_daemon_isolation_qubes_whonix.po:84]: The flags comments should be on their own lines.

File: _i18n/en/resources/user-guides/weblate/cli_wallet_daemon_isolation_qubes_whonix.pot~ (hunk 2 above)
Comment (wirrickelliot) [inline _i18n/en/resources/user-guides/weblate/cli_wallet_daemon_isolation_qubes_whonix.pot~:1]: I don't believe this file should be here.

File: _i18n/it/resources/user-guides/weblate/cli_wallet_daemon_isolation_qubes_whonix.po~ (hunk 3 above)
Comment (wirrickelliot) [inline _i18n/it/resources/user-guides/weblate/cli_wallet_daemon_isolation_qubes_whonix.po~:1]: I don't believe this file should be here either.

File: _i18n/en/resources/user-guides/weblate/cli_wallet_daemon_isolation_qubes_whonix.pot [hunk 8]
```diff
@@ -0,0 +1,251 @@
...
+# This file is distributed under the same license as the monero-site repository.
+#
```
Comment (wirrickelliot) [inline _i18n/en/resources/user-guides/weblate/cli_wallet_daemon_isolation_qubes_whonix.pot:4]: The headers aren't uniform between all of the files.

File: _i18n/en/resources/user-guides/weblate/cli_wallet_daemon_isolation_qubes_whonix.pot (hunk 4 above)
Comment (wirrickelliot) [inline _i18n/en/resources/user-guides/weblate/cli_wallet_daemon_isolation_qubes_whonix.pot:5]: This should be unfuzzied.

File: _i18n/it/resources/user-guides/weblate/cli_wallet_daemon_isolation_qubes_whonix.po (hunk 6 above)
Comment (wirrickelliot) [inline _i18n/it/resources/user-guides/weblate/cli_wallet_daemon_isolation_qubes_whonix.po:45]: This is still present in multiple files.

File: _i18n/fr/resources/user-guides/weblate/cli_wallet_daemon_isolation_qubes_whonix.po (hunk 7 above)
Comment (wirrickelliot) [inline _i18n/fr/resources/user-guides/weblate/cli_wallet_daemon_isolation_qubes_whonix.po:84]: The flags comments should be on their own lines.

**Review by wirrickelliot — CHANGES_REQUESTED** [review]

**Comment by erciccione** [comment]
The headers weren't a problem, but they are really useless to us and doesn't worth spend time to personalize them, so i just used the standard version everywhere. Fixed the fuzzy strings and uniformed the files.

**Comment by wirrickelliot** [comment]
I mainly just mentioned the headers because you changed them for some files but not for others.

Should these two files not be deleted?
- `_i18n/en/resources/user-guides/weblate/cli_wallet_daemon_isolation_qubes_whonix.pot~`
- `_i18n/it/resources/user-guides/weblate/cli_wallet_daemon_isolation_qubes_whonix.po~`

**Comment by erciccione** [comment]
> I mainly just mentioned the headers because you changed them for some files but not for others.

You were right. Now i switched to the standard one, so the problem is avoided :)

> That's what I meant when I said "I don't believe this file should be here." Sorry if that was unclear.

Sorry, i misunderstood. Yes those shouldn't be there. Deleting them/

File: _i18n/tr/resources/user-guides/weblate/cli_wallet_daemon_isolation_qubes_whonix.po [hunk 9]
```diff
@@ -0,0 +1,252 @@
+# SOME DESCRIPTIVE TITLE
+# Copyright (C) YEAR Free Software Foundation, Inc.
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+#, fuzzy
```
Comment (wirrickelliot) [inline _i18n/tr/resources/user-guides/weblate/cli_wallet_daemon_isolation_qubes_whonix.po:6]: I just noticed this was here also.

**Comment by wirrickelliot** [comment]
Sorry, one last thing: are the changes to `_i18n/it/resources/user-guides/cli_wallet_daemon_isolation_qubes_whonix.md` intentional? It seems to be the only file with a structure different from the rest.

File: _i18n/tr/resources/user-guides/weblate/cli_wallet_daemon_isolation_qubes_whonix.po (hunk 9 above)
Comment (erciccione) [reply _i18n/tr/resources/user-guides/weblate/cli_wallet_daemon_isolation_qubes_whonix.po:6]: Fixed. Thanks :)

**Comment by erciccione** [comment]
It's the result of a test (conversion `.po` > `.md`). po4a wraps the text when converting. I left it because there is a string translated and in any case doesn't really matter, users won't touch these files anymore.

**Review by wirrickelliot — APPROVED** [review]

**Pull request merged**